mrbgems/mruby-print/src/print.c

```diff
@@ -13,7 +13,7 @@ printstr(mrb_state *mrb, mrb_value obj)
 {
   if (mrb_string_p(obj)) {
 #if defined(_WIN32)
-    if (isatty(fileno(stdout))) {
+    if (_isatty(_fileno(stdout))) {
```

Comment:
I don't have strong opinion. However, I prefer to use non-underbar-ed name APIs even though the codes runs only on the Windows (if it's possible).